TorqueDataConnect/include/TorqueDataConnectConfig.php

```diff
@@ -5,7 +5,7 @@ class TorqueDataConnectConfig {
   public static function convertPageToColumnConfig($page) {
     $title = Title::newFromText($page);
     if(!$title->exists()) {
-      array_push(self::$errors, $page . " does not exist, but is reference in config as a column config.");
+      array_push(self::$errors, wfMessage("convert-column-page-ne", $page)->parse());
```

Comment:
This is more a question for the next reviewer (presumably @frankduncan) than it is a comment -- I'm assuming that the next reviewer has both more i18n experience and more PHP experience than I have :-).

Is `wfMessage()` a built-in thing that we automatically get just by virtue of using PHP?  It automatically knows to look in `i18n/<two-letter-ISO-639-1-language-code>.json` for the strings for a given localization?

I think I came in expecting the change to be larger.  I thought there'd be an import of some i18n library or something -- but maybe that's not necessary if a perfectly fine one is built in already.  (I also sort of expected there to be more strings that would need replacing!  But @frankduncan is better positioned than I am to know how many strings need to be internationalized in Torque.)

Reply:
`wfMessage()` is one of the MediaWiki i18n functions, and yes, it knows how to look in all the right places as configured by your plugin in extension.json